### ISSUE TITLE
Bad Lag

### ISSUE THREAD

**Issue by jmdl1983** [OgarProject/Ogar#613]
I have installed this on a Windows 7 machine with quad-core processor and 16GB RAM, but the lag with even 2 players on a LAN is ungodly.

I have also installed this on a Server 2012 machine with eight-core processor and 64GB RAM, with the same result.  And going over WAN is by far worse.

What am I missing here?

1, I see notes that the java runs in a single core process, so multi-cored processors aren't really that advantageous.

2, I have tried increasing and decreasing board size.

3, I have even disabled the stats server, and logging thinking perhaps it was overtaxing resources.

4, I have monitored RAM usage, and it never even hits 1GB and processor usage never even hits 5%.

5, I have tried this on SSD, HDD, RAID HDD, and even a striped RAID SSD.

Does this just always lag bad on a Windows platform?  Do I need to load inside of a linux machine?

Here is my current config file:
`serverMaxConnections = 10
serverPort = 499
serverGamemode = 0
serverBots = 4
serverViewBaseX = 1920
serverViewBaseY = 1080
serverStatsPort = 88
serverStatsUpdate = 60
serverLogLevel = 0
serverScrambleCoords = 1
serverScrambleMinimaps = 1
serverTeamingAllowed = 1
serverMaxLB = 10

borderLeft = 0
borderRight = 6000
borderTop = 0
borderBottom = 6000

spawnInterval = 20
foodSpawnAmount = 10
foodStartAmount = 100
foodMaxAmount = 500
foodMass = 1
foodMassGrow = 1
foodMassGrowPossiblity = 50
foodMassLimit = 5
foodMassTimeout = 120
virusMinAmount = 10
virusMaxAmount = 50
virusStartMass = 100
virusFeedAmount = 7

ejectMass = 13
ejectMassCooldown = 100
ejectMassLoss = 15
ejectSpeed = 100
ejectSpawnPlayer = 50

playerStartMass = 10
playerBotGrowEnabled = 1
playerMaxMass = 22500
playerMinMassEject = 32
playerMinMassSplit = 36
playerMaxCells = 16
playerRecombineTime = 30
playerMassAbsorbed = 1.0
playerMassDecayRate = .002
playerMinMassDecay = 9
playerMaxNickLength = 15
playerSpeed = 35
playerDisconnectTime = 60

tourneyMaxPlayers = 12
tourneyPrepTime = 10
tourneyEndTime = 30
tourneyTimeLimit = 20
tourneyAutoFill = 0
tourneyAutoFillPlayers = 1`

**Comment by Barbosik** [comment]
try MultiOgar, it's much faster because it uses new physics engine with vanilla physics: https://github.com/Barbosik/MultiOgar/

**Comment by David1ali12** [comment]
@Barbosik  is it like agario physics 100%?

**Comment by Barbosik** [comment]
yes, all movements, split, eject the same as on vanilla servers.
Currently there are some issues:
* no food grow, it's just spawned with random size;
* if you has 1 big blob and 14 small blobs, split may be applied to small blobs
* if there are a lot of blobs involved in collision, they may jitter, you can fix it by un-commenting loop for resolve rigid body collision, but it may leads to bad performance
